system/athena/athenad.py

```diff
@@ -762,13 +762,25 @@ def ws_manage(ws: WebSocket, end_event: threading.Event) -> None:
       onroad_prev = onroad
 
       if sock is not None:
```

Comment:
Does it work if you check the platform instead?
```
if sys.platform != "win32" and sys.platform != "darwin":
```
That's how it's setup where the types themselves are defined https://github.com/python/typeshed/pull/10530/files#diff-8d751cc8017e3c1cde34b01bdd3e44d5ba963d53a47c6262972d206200d4913fL178